Subject: [PATCH] Add custom image style sizes in ckeditor 5 toolbar
---
Index: core/modules/ckeditor5/ckeditor5.ckeditor5.yml
<+>UTF-8
===================================================================
--- ckeditor5.ckeditor5.yml	2025-04-30 13:26:18
+++ ckeditor5-patch	2025-04-30 13:27:03
@@ -621,6 +621,22 @@
           -
             name: 'resizeImage:original'
             value: null
+          -
+            name: 'resizeImage:small'
+            value: '360'
+            label: 'Small'
+          -
+            name: 'resizeImage:small'
+            value: '480'
+            label: 'Medium'
+          -
+            name: 'resizeImage:small'
+            value: '640'
+            label: 'Large'
+          -
+            name: 'resizeImage:small'
+            value: '768'
+            label: 'Very Large'
         toolbar: [resizeImage]
   drupal:
     label: Image resize
